### PR TITLE
add useless lines 

### PR DESCRIPTION
**Pull request by jinohschool**

### CODE REVIEW

File: another.c [hunk 1]
```diff
@@ -1 +1,2 @@
-another file from tutorial
```
Comment (jinoh808) [inline another.c:1]: fantastic edit good job!

File: another.c [hunk 2]
```diff
@@ -1 +1,2 @@
...
\ No newline at end of file
+another file from tutorial
+editted by contributer
```
Comment (jinoh808) [inline another.c:2]: what a poor code you idiot! 
fall to hell!! and never get back!!

**Review by jinoh808 — CHANGES_REQUESTED** [review]
check my coments

File: another.c (hunk 2 above)
Comment (jinohschool) [reply another.c:2]: are you serious?

**Review by jinoh808 — APPROVED** [review]
lgtm

**Pull request merged**